Week3/homework/3-step3.js

```diff
@@ -1,25 +1,49 @@
-'use strict';
+"use strict";
 
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
-  // add your code here
-  return str;
+  if (num <= 0) return "";
```

Comment:
If your `if` statement return, there is no need for an `else`.

Also, it is best to always use curly braces for an `if` block:

```js
if (num <= 0) {
  return "";
}
```

However, you do not need the `if` statement at all, because the loop body will not be executed if `num` is less or equal to zero:

```js
function repeatStringNumTimesWithFor(str, num) {
  let result = "";
  for (let i = 0; i < num; i++) {
    result += str;
  }
  return result;
}
```